includes/class-sensei-groups-landing-page.php

```diff
@@ -94,13 +94,13 @@ public function display_student_groups_landing_page() {
 			<div class="sensei-promo-groups__actions">
 				<a
 				class="button button-primary sensei-promo-groups__primary-action"
-				href="<?php echo esc_url( 'https://senseilms.com/pricing/' ); ?>"
+				href="<?php echo esc_url( 'https://senseilms.com/sensei-pro/?utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=groups' ); ?>"
 				target="_blank"
 				>
 				<?php echo esc_html( __( 'Get Sensei Pro', 'sensei-lms' ) ); ?>
 			</a>
 			<a	class="sensei-promo-groups__secondary_action"
-				href="<?php echo esc_url( 'https://senseilms.com/pricing/' ); ?>"
+				href="<?php echo esc_url( 'https://senseilms.com/sensei-pro/?utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=groups' ); ?>"
```

Comment:
```suggestion
				href="<?php echo esc_url( 'https://senseilms.com/sensei-pro/#features?utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=groups' ); ?>"
```

Reply:
Today both links are going to the same place.
The second link should go to https://senseilms.com/sensei-pro/?utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=groups#features 

(Just an extra #features to go to the correct section)

Reply:
Updated in f0406721a355f02843600d571143bf1226e2965c.